<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTSAnnotator</class>
 <widget class="QWidget" name="TTSAnnotator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>508</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="totalTranscriptWordsLbl">
         <property name="text">
          <string>Transcript words: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalTranscriptEditedWordsLbl">
         <property name="text">
          <string>Transcript words Edited: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalMispronouncedWordsLbl">
         <property name="text">
          <string>Mispronounced Words: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalMispronouncedEditedWordsLbl">
         <property name="text">
          <string>Mispronounced Edited Words: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalTaggedWordsLbl">
         <property name="text">
          <string>Tagged Words: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalTaggedEditedWordsLbl">
         <property name="text">
          <string>Tagged Words Edited: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="InsertRowButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="statusTip">
          <string extracomment="Disabled"/>
         </property>
         <property name="whatsThis">
          <string extracomment="Disabled"/>
         </property>
         <property name="text">
          <string>Insert a row</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteRowButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string extracomment="Disabled">Remove selected row</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAsTableButton">
         <property name="text">
          <string>Save As</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveTableButton">
         <property name="text">
          <string>Save </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
